plugins/scripts/create_desktop_file.py

```diff
@@ -22,16 +22,28 @@ def create_desktop_file(
     destination=None,
     icon=None,
     terminal=None,
+    junogl=None,
 ):
     """Create a desktop file for the application based on the provided arguments."""
+
+    # update terminal option - donest break current runs!
+    # desktop files need to be lowercase true or false
     if terminal == "True":
-        terminator = "terminator -x "
+        terminal_run = "true"
+    else:
+        terminal_run = "false"
+
```

Comment:
Looks good, but would it be easier to have terminal default to false, and then just do terminal_run = terminal.lower()?

Reply:
we could do that, i tried but we end up doing the same since all the installer now do uper False.
until i do a pass on arguments on all installer which make desktop shorctut this needs to stay as is...